packages/xo-server/src/api/vm.mjs

```diff
@@ -257,6 +264,12 @@ create.params = {
     optional: true,
   },
 
+  createVtpm: {
+    type: 'boolean',
+    optional: true,
```

Comment:
You don't have to explicitly mark it as optional if you provide a default value :slightly_smiling_face: